src/services/flowsheet.service.ts

```diff
@@ -234,6 +304,11 @@ export const getLatestShow = async (): Promise<Show> => {
   return latest_show;
 };
 
+export const getMostRecentTwoShows = async (): Promise<Show[]> => {
```

Comment:
Any reason not to parameterize this for last n shows?

Reply:
Nope - updated!